docs/application/native/guides/multimedia/image-edit.md

```diff
@@ -82,214 +78,348 @@ To enable your application to use the image util functionality:
 
    For more information on the YUV color space, see [http://www.fourcc.org/yuv.php](http://www.fourcc.org/yuv.php).
 
-4. To support the `image_util_transform_run()` function, which is used for all image transformations, set the source image and create a handle for it (to be used as the second parameter):
+4. To support `image_util_transform_run()`, which is used for all image transformations, set the source image and create a handle for it (to be used as the second parameter):
 
    ```
    ret = image_util_decode_create(&decode_h);
    ret = image_util_decode_set_input_path(decode_h, SAMPLE_FILENAME);
    ret = image_util_decode_set_colorspace(decode_h, colorspace);
-   ret = image_util_decode_set_output_buffer(decode_h, &img_source);
 
-   ret = image_util_decode_run(decode_h, &width, &height, &size_decode);
+   ret = image_util_decode_run2(decode_h, &src_image);
    ret = image_util_decode_destroy(decode_h);
+   ```
+
+<a name="image_colorspace"></a>
+## Converting Image Color Space
+
+To convert an image from one color space of an image to another:
+
+1. Create a transformation handle using `image_util_transform_create()`:
+
+   ```
+   transformation_h transform_h;
+   ret = image_util_transform_create(&transform_h);
+   ```
+
+2. Set the target color space using `image_util_transform_set_colorspace()`:
+
+   ```
+   ret = image_util_transform_set_colorspace(transform_h, colorspace);
+   ```
+
+3. Execute the transformation using `image_util_transform_run2()`:
+
+   ```
+   ret = image_util_transform_run2(transform_h, src_image, &dst_image);
+   ```
+
+4. Execute the transformation using `image_util_transform_run2_async()`:
 
-   ret = media_format_create(&fmt);
-   ret = media_format_set_video_mime(fmt, colorspace);
-   ret = media_format_set_video_width(fmt, width);
-   ret = media_format_set_video_height(fmt, height);
+   ```
+   ret = image_util_transform_run2_async(transform_h, src_image,
+                                        (image_util_transform_completed2_cb)completed_callback,
+                                        user_data);
+   ```
+
+   > **Note**
+   >
+   > - `image_util_transform_run2()` and `image_util_transform_run2_async()` only converts the color space. These functions do not change the image width, height, or any other image property. Due to these restrictions of the image processing library, not all color space combinations are supported for conversion.
+
+5. Handle the transformation results in the `image_util_transform_completed2_cb()` callback, which is invoked after the transformation is complete.
```

Comment:
Global for callback: Suggest removing the and callback. Here in the `image_util_transform_completed2_cb(), cb denotes callback. Therefore, suggest rephrasing to:

 Handle the transformation results in `image_util_transform_completed2_cb()`, which is invoked after the transformation is complete.

Reply:
okay, I will remove "the" and "callback" in case of "the *** callback"